store/blockstore.go

```diff
@@ -0,0 +1,122 @@
+package store
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/gob"
+	"sync"
+
+	"github.com/lazyledger/optimint/types"
+	"go.uber.org/multierr"
+	"golang.org/x/crypto/sha3"
+)
+
+var (
+	blockPreffix = [1]byte{1}
+	indexPreffix = [1]byte{2}
```

Comment:
Add a comment describing what these prefixes are used for.
Also 
```suggestion
	blockPrefix = [1]byte{1}
	indexPrefix = [1]byte{2}
```